pkg/kube/client.go

```diff
@@ -105,7 +105,7 @@ func (c *Client) IsReachable() error {
 	client, err := c.getKubeClient()
 	if err == genericclioptions.ErrEmptyConfig {
 		// re-replace kubernetes ErrEmptyConfig error with a friendy error
-		// moar workarounds for Kubernetes API breaking.
+		// more workarounds for Kubernetes API breaking.
```

Comment:
Yes, it's cute. No, I don't think it adds value. Happy to drop.

Reply:
I would strongly disagree that it adds value. Please revert.